packages/mongodb-chatbot-server/src/routes/conversations/addMessageToConversation.ts

```diff
@@ -328,7 +328,7 @@ async function sendStaticResponse({
       data: apiRes.content,
     });
 
-    if (references) {
+    if (references && references.length > 0) {
```

Comment:
should this be tested somehow?

Reply:
Maybe? It's definitely correct in the intended use of the references event but we could codify that in a test. However we don't have unit tests for `sendStaticResponse` atm and it'll take some work to set that up so I'll make a separate ticket.

Reply:
https://jira.mongodb.org/browse/EAI-411